core-java-modules/core-java-concurrency-advanced-6/README.md

```diff
@@ -0,0 +1,3 @@
+
+### Relevant Articles:
+- [Round Robin and AtomicInteger in Java](https://www.baeldung.com/round-robin-and-atomicinteger-in-java)
```

Comment:
We can remove this for now, since we won't know the final link until the article is published.

Reply:
surely, the following commits are targeted to resolve this.